.travis.yml

```diff
@@ -6,11 +6,11 @@ cache:
 git:
   depth: 3
 stages:
-- name: pr-previews
-  if: "(branch = master AND type = pull_request) OR branch != master"
+- name: testing
+  if: branch = master AND type = pull_request
```

Comment:
I just realized that this won't build PRs that don't target master, so this won't work. What was the reason we couldn't just do `type = pull_request` again? I forgot how we came to that conclusion.